### PR TITLE
blogpost on 2017 All HC Developers Meetup

### PR DESCRIPTION
**Pull request by cosimalaube**
Content has been already reviewed by two colleagues.

What is open for review & test: layout of article/display of the embedded images (as I still run Win7 and haven't fiddled with setting up local blog instance yet).

### CODE REVIEW

File: _posts/2017-07-14_hc-all-devs-meetup.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+> Open Space creates a stable, methodic frame, where many people can work on their issues self-organized and self-responsible. The goal is to work on essential, innovative and solution focused subjects in a short time with a big number of people.
+
+## ... and awesome outcomes for real work-life
+
+<img src="{{site.baseurl}}/img/posts/2017-07-14_hc-all-devs-meetup/oneOfTheSessions.jpg" alt="one of the sessions at a restaurant" width="200" style="float: right; margin: 1rem;" />
+
+We had over 40 sessions for both days from discussions to live coding and also creating & incubating new ideas and actions. For example a couple of Communities of Practise (CoP) e.g. on Scala or Testing were started or promoted by these intense days at the beach. Also this tech blog you are currently reading had been reanimated - or to be more precise: rebuilt - as a direct consequence of the gathering.
```
Comment (gilligan) [inline _posts/2017-07-14_hc-all-devs-meetup.md]: Just a little detail but I would suggest to change this:

> Also this tech blog you are currently reading had been reanimated - or to be more precise: rebuilt - as a direct consequence of the gathering

to this:

> The reboot of this very blog was another project that was started during the meetup.

I find the current sentence somewhat long-winded. And `had been` is used to refer to time spans so its use seems a bit off to me here. 

Just my 2cents ;)

File: _posts/2017-07-14_hc-all-devs-meetup.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+The days after the meetup you could hear fun stories and memories being shared almost everywhere. And the weeks after the teams and people earned the fruits of this investment our company made for us: people started travelling more often to other locations, distributed teams started working co-located for a period to get stuff done even better.. and now you simply knew whom to ask for what. One of my colleagues told me with a twinkle in his eye that the FPM rate (flames per minute) on a hot-topic Slack channel reduced to almost zero and the former flaming was replaced by constructive and valuable exchange.
+
+For me as a former software engineer who 'went Scrum master' now it was amazing to see that successful gathering happen. I enjoyed connecting with many different people, learning what my colleagues are technically doing, hearing about painpoints as well as our treasures and simply having fun together.
```
Comment (gilligan) [inline _posts/2017-07-14_hc-all-devs-meetup.md]: 'went Scrum master' ? Why not just smth like "who decided to become a Scrum master" or so.

File: _posts/2017-07-14_hc-all-devs-meetup.md (hunk 1 above)
Comment (cosimalaube) [inline _posts/2017-07-14_hc-all-devs-meetup.md]: Thanks. Integrated your suggestion. It's way better than my sentence ;)

File: _posts/2017-07-14_hc-all-devs-meetup.md (hunk 2 above)
Comment (cosimalaube) [inline _posts/2017-07-14_hc-all-devs-meetup.md]: Thanks for your suggestion. Here I like to stick to my own writing style.

**Comment by wolframkriesing** [comment]
@cosimalaube if you want to add a link to https://twitter.com/search?f=tweets&q=%23hcalldevs17 that might underline, that we love to also talk about it ... or simply share :)

**Pull request merged**